package.json

```diff
@@ -5,11 +5,37 @@
   "description": "{{project_name}} frontend assets.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
Might be a good idea to also add a watch command.
ie: `"test:watch": "npm test -- --watch",`